core/txbuilder/finalize.go

```diff
@@ -45,6 +45,9 @@ func FinalizeTx(ctx context.Context, c *protocol.Chain, s Submitter, tx *legacy.
 	if err != nil {
 		return errors.Wrap(err, "tx rejected")
 	}
+	if tx.Tx.MaxTimeMs > 0 && tx.Tx.MaxTimeMs < c.TimestampMS() {
```

Comment:
Is this part absolutely necessary? I'd prefer to keep change to point releases as small and conservative as possible.